react/rules/DEU.js

```diff
@@ -37,6 +37,7 @@ export default {
       label: 'birthDate',
       type: 'date',
       validate: isPastDate,
+      mask: (value) => msk.fit(value, '99.99.9999'),
```

Comment:
I don't see the `msk` imported anywhere. Wouldn't this be a cause for a reference error?

Reply:
Done!